.travis.yml

```diff
@@ -9,6 +9,7 @@ python:
 - '3.6-dev'
 env:
 - TOXENV=py-test
+- SELFINSTALL=1
```

Comment:
Maybe consider instead just having two TOXENVs, one which is py-test and the other self-install... and perform the self-install checks in that toxenv.